### ISSUE TITLE
Free OTP crashes when importing QR with special characters in Account Name

### ISSUE THREAD

**Issue by SaSue** [freeotp/freeotp-ios#85]
If a special characters in Account Name, FreeOTP for iOS crashes and it is not longer possible to start FreeOTP. (crashes every time again when start FreeOTP)
Tested with iPhone6 and 7plus, with iOS 10.3.2 and 10.3.3 Beta; FreeOTP Android works fine for same QR-Code

It is reproducible with FreeOTP QR Code generator at https://freeotp.github.io/qrcode.html (see att. Screenshot)
<img width="649" alt="screenshot 2017-07-03 14 47 28" src="https://user-images.githubusercontent.com/11920816/27793729-86d0d4e0-5fff-11e7-929a-52102a2b5f3c.png">

**Comment by 9px** [comment]
Yes. I had same issue in ios as well. when i scan a QR barcode with SHA512 hash and 8 digits, the free otp does not add account and the app crashes every time again when start FreeOTP.

**Comment by TBOTO** [comment]
I'm having same issue in IOS 11.3.1 iPhone 6S. Now app is in constant start-up crash.
Please fix as soon as possible as I need to access codes.
Thank you

**Comment by antonini** [comment]
Same issue with an iPhone 7 with iOS 11.3.1.

**Comment by feliperalmeida** [comment]
This bug seems to no longer exist on ```master``` branch, but the App Store version was last updated on Aug 1, 2014. Is it possible for the maintainers to push a new build to the App store? That would be awesome. @npmccallum

**Comment by corsch** [comment]
Is there a new release planned or at least a workaround to remove the bad characters?

**Comment by xyb** [comment]
@npmccallum My FreeOTP app keep crash after added Chinese characters in issue name, I would be grateful if you could release a new version app.

**Comment by ifcwlme** [comment]
I have same issue for Chinese in issuer name.

**Comment by bliemli** [comment]
Same issue here after scanning a QR code from Microsoft.

**Comment by einar-io** [comment]
Same issue here.

**Comment by justin-stephenson** [comment]
Hello,

This issue should be resolved in `FreeOTP v2.0`, can someone please confirm that? We also added unit tests to ensure this issue does not happen going forward.

**Comment by rgloor** [comment]
Would there be a chance, that somebody uploads the newer version to Android store?
There is only version 1.5 available.
Thanks in advance.

**Comment by justin-stephenson** [comment]
> Would there be a chance, that somebody uploads the newer version to Android store (Google Play Store)? There is only version 1.5 available. _It looks as Apple-Store has newest version._ Thanks in advance.

The 2.0 Beta is available in the Play store if you would like to try it out.
